src/authorizationProvider.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import authService from "./services/authService";
+
+const refreshToken = 'RefreshToken';
+const accessToken = 'AccessToken';
+const AuthorizationContext = React.createContext();
+class AuthorizationProvider extends Component {
+    state = {
+        loggedIn: null,
+        userFirstName: null,
+        userLastName: null,
+        userRole: null
+    }
+
+    async componentDidMount() {
+        if (localStorage.getItem(accessToken) !== 'null' &&
```

Comment:
kedy moze byt v localstorage `"null"` string?

Reply:
neviem.. ale z nejakeho dovodu sa mi to stava.. musim este investigovat